thinc/layers/softmax.py

```diff
@@ -17,29 +17,58 @@ def Softmax(
     nI: Optional[int] = None,
     *,
     init_W: Callable = zero_init,
-    init_b: Callable = zero_init
+    init_b: Callable = zero_init,
 ) -> Model[InT, OutT]:
     return Model(
         "softmax",
         forward,
         init=partial(init, init_W, init_b),
         dims={"nO": nO, "nI": nI},
         params={"W": None, "b": None},
+        attrs={"normalize_softmax": True},
+    )
+
+
+@registry.layers("Softmax.v2")
+def Softmax_v2(
+    nO: Optional[int] = None,
+    nI: Optional[int] = None,
+    *,
+    init_W: Callable = zero_init,
+    init_b: Callable = zero_init,
+    normalize_outputs: bool = True,
```

Comment:
Wonder whether `v2` should also have `temperature`. I usually think of `softmax` having temperature, but not sure if its important to have it in the implementation. Just wondering, because in `Pytorch` users can add it easily, but in `thinc` I feel like users have to deal with the derivative themselves and that might not be that nice. Not sure.

Reply:
I think it would be nice to have temperature! I think it should be a separate PR, but we could add this to v2 before cutting a new release?

Related: it would also be nice to have label smoothing in the cross entropy loss.

Reply:
Okok, let's have maybe two separate small PRs then for label-smoothing and temperature!